csp_billing_adapter_local/plugin.py

```diff
@@ -53,7 +56,10 @@ def get_local_path(filename: str):
 
 @csp_billing_adapter.hookimpl
 def setup_adapter(config: Config):
+    logging.basicConfig(datefmt='%Y-%m-%dT%H:%M:%S')
```

Comment:
Logging is already being setup in the main adapter in the main loop:
https://github.com/SUSE-Enceladus/csp-billing-adapter/blob/573cd6aaa4505aa9146b66752d61c31c36a9da68/csp_billing_adapter/adapter.py#L387

This is called before the setup_adapter() hook is being called, so perhaps this implementation should be written to modify the existing setup, rather than do an initial setup.

Reply:
Yes, I'm aware but the logging properties set on the core adapter are not preserved on the local one
 I updated it to use the same date format as the core adapter